index.js

```diff
@@ -176,25 +176,29 @@ function redisStore(args) {
    * @param {Function} cb - A callback that returns a potential error and the response
    */
```

Comment:
Ad `@returns {Promise}` to have proper function documentation and it is known a promise is returned.